app/views/_includes/candidate-feedback.njk

```diff
@@ -13,7 +13,7 @@
     </h2>
   {% endif %}
 
-  <p>Here’s why your application was unsuccessful.</p>
+  <p>Here’s why your application was unsuccessful on this occasion.</p>
```

Comment:
Emma and I talked about it and felt this wasn't needed. It also complicates matters because of the ‘other choice} question.